Applications/AlexNet/jni/cifar_dataloader.cpp

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jihoon Lee <jhoon.it.lee@samsung.com>
+ *
+ * @file   cifar_dataloader.h
+ * @date   24 Jun 2021s
+ * @brief  dataloader for cifar
+ * @see    https://github.com/nnstreamer/nntrainer
+ * @author Jihoon Lee <jhoon.it.lee@samsung.com>
+ * @bug    No known bugs except for NYI items
+ */
+
+#include "cifar_dataloader.h"
+
+#include <cstring>
+#include <iostream>
+#include <nntrainer_error.h>
+#include <random>
+
+namespace nntrainer::alex {
+
+namespace {
+
+/**
+ * @brief fill label to the given memory
+ *
+ * @param data data to fill
+ * @param length size of the data
+ * @param label label
+ */
+void fillLabel(float *data, unsigned int length, unsigned int label) {
```

Comment:
This Cifar 100 data loader and Random data loader are used quite a lot through the applications. It might be better to merge one in application utils and shared. I made the issues here. #1969

Reply:
#1969 
FIX in Application {Alexnet, VGG, Resnet}